src/app/components/Wallet.tsx

```diff
@@ -84,58 +92,93 @@ class Wallet extends Component<WalletProps, WalletState> {
      */
     public render(): ReactNode {
         return (
-            <div className="wallet col">
-                {this.state.isBusy && (
-                    <Spinner />
-                )}
-                {(!this.state.wallet || !this.state.wallet.seed) && (
-                    <div className="card">
-                        <div className="card--header">
-                            <h2>Wallet</h2>
+            <div
+                className={`wallet col ${(this.state.walletServiceLoaded && (!this.state.wallet || !this.state.wallet.seed)) ? "wallet--homepage" : "relative center fill"}`}>
```

Comment:
`relative center fill` are new classes that center the content of the wallet, but then settings is not centered and it looks jumping. People like to use the wallet on full screens and then we have this
![image](https://user-images.githubusercontent.com/3624944/119864568-0a22b100-bf1b-11eb-94fb-2328805021ee.png) 
![image](https://user-images.githubusercontent.com/3624944/119864591-0f7ffb80-bf1b-11eb-8ac4-bbdc7973bfac.png)

Reply:
yup, that `center` class centers the content

Reply:
yeah it was happening on large screens. Fix pushed